src/components/MovieTile/MovieTile.js

```diff
@@ -0,0 +1,77 @@
+import React, {useState} from 'react'
+import PropTypes         from 'prop-types'
+import styles            from './MovieTile.module.css'
+import Button            from '@mui/material/Button'
+import Menu              from '@mui/material/Menu'
+import MenuItem          from '@mui/material/MenuItem'
+import {
+    usePopupState,
+    bindTrigger,
+    bindMenu,
+}                        from 'material-ui-popup-state/hooks'
+import EditIcon          from '@mui/icons-material/Edit'
+import DeleteIcon        from '@mui/icons-material/Delete'
+import MoreVertIcon      from '@mui/icons-material/MoreVert';
+import {Grid}            from "@mui/material";
+
+const GENRE_DELIMITER = ', '
+
+const MovieTile = ({imageUrl, title, releaseYear, genres}) => {
+    const [genreRow, _] = useState(genres.join(GENRE_DELIMITER))
+    const popupState    = usePopupState({variant: 'popover', popupId: 'demoMenu'})
+    const menuBtnStyles = {
+        backgroundColor: '#2A202D',
+        width          : '36px',
+        minWidth       : '36px',
+        height         : '36px',
+        borderRadius   : '50%'
+    }
+    return (
+        <Grid className={styles.MovieTile}

```

Comment:
for classname camelCase is used usually, pls change it to 
```suggestion
        <Grid className={styles.movieTile}

```